tests/Unit/Services/AiWorkflowContextTest.php

```
@@ -57,6 +57,18 @@ public function resilientOptionsOptIntoOpenAiFallback(): void
         ], $options);
     }
 
+    #[Test]
+    public function resilientOptionsDisableOpenAiFallbackWithoutPositiveTenant(): void
+    {
+        $options = AiWorkflowContext::resilientOptions('work_item_generation', '4.2', 3200);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Services\AiWorkflowContext' in method 'resilientOptionsDisableOpenAiFallbackWithoutPositiveTenant'. (undefined)

(StaticAccess)